src/interfaces/index.ts

```diff
@@ -424,7 +424,6 @@ export interface PassengerDetails {
 
 export interface CompanionInfo {
     passengerType: string;
-    passengerTypeName?: string;
```

Comment:
We also ought to remove the GroupPassengerInfo interface below this, as it is not used anywhere.